RecoHGCal/TICL/plugins/TrackstersPCA.cc

```diff
@@ -52,10 +58,22 @@ void ticl::assignPCAtoTracksters(std::vector<Trackster> &tracksters,
       fillPoint(layerClusters[trackster.vertices(i)], weight);
       for (size_t j = 0; j < 3; ++j)
         barycenter[j] += point[j];
+
+      // Also compute timing
+      if ((usedLC.insert(trackster.vertices(i))).second) {
+        float timeE = layerClustersTime.get(trackster.vertices(i)).second;
+        if (timeE > -1.) {
```

Comment:
Shouldn't this also be `if (timeE > 0.f)` ?

Reply:
Done: https://github.com/cms-sw/cmssw/pull/32046/commits/0309768d2851d4cbfb13931fe980e57f2b33fe1a